### DISCUSSION TITLE
2021/8/30 倍速播放、快进，在bili无法运行

### DISCUSSION THREAD

**Discussion by kinchung7** [xxxily/h5player#299] [General]
重新安装也无法解决

**Comment by luguoyixiazi** [comment]
你在2787行下面添加list.push(document.getElementsByTagName('bwp-video')[0])
直接拿video标签拿的有问题，哔哩哔哩封装了一层，直接获取video标签有问题，具体原理我还没搞清楚，可能是生命周期之类的或者是运行机制？反正异步也会报错promise失败，但是拿bwp-video是可以操作到哔哩哔哩视频的
